modules/checking-mco-node-status.adoc

```diff
@@ -237,3 +251,17 @@ status:
 <1> The `MachineConfigNode` object name.
 <2> The new machine configuration. This field updates after the MCO validates the machine config in the `UPDATEPREPARED` phase, then the status adds the new configuration.
```

Comment:
I'm not sure if you want to address the CQA 2.0 guidance in this PR or not but callouts are present in multiple modules.